cli/internal/core/engine.go

```diff
@@ -107,7 +104,8 @@ func (e *Engine) getTaskDefinition(pkg string, taskName string, taskID string) (
 	if task, ok := e.Tasks[taskName]; ok {
 		return task, nil
 	}
-	return nil, errNoTask
+
+	return nil, fmt.Errorf("Missing %v task definition in turbo.json", taskID)
```

Comment:
Is it beneficial at all to include the `taskName` that wasn't found?

Reply:
Probably if you're running multiple tasks

Reply:
good call, added both